### ISSUE TITLE
Add Canada.ca template pages

### ISSUE THREAD

**Issue by yileifeng** [ramp4-pcar4/highcharts-editor#151]
## Description

HACK is missing a Canada.ca template page to match other web mapping applications. Critical for https://github.com/ramp4-pcar4/custom-projects/issues/340. Update Vite build configuration to support index-en-ca and index-fr-ca entry points. Ensure that all HACK styling remains consistent when residing inside Canada.ca template.
